scipy/signal/_spectral_py.py

```diff
@@ -57,7 +71,19 @@ def lombscargle(x,
     Raises
     ------
     ValueError
-        If the input arrays `x` and `y` do not have the same shape.
+        If any of the input arrays x, y, freqs, or weights are not 1D.
+    ValueError
+        If the input arrays x, y, and weights do not have the same shape.
```

Comment:
These two can be combined for conciseness

Reply:
Done.